do.py

```diff
@@ -0,0 +1,210 @@
+import fnmatch
+import os
+import re
+import sys
+import shutil
+import subprocess
+
+
+def setup():
+    run(
+        [
+            py() + " -m pip install --upgrade pip",
+            py() + " -m pip install --upgrade virtualenv",
+            py() + " -m virtualenv .env",
+        ]
+    )
+
+
+def init():
+    run(
+        [
+            py() + " -m pip install -r requirements.txt",
+        ]
+    )
+
+
+def lint():
+    paths = ["tests", "do.py"]
+
+    run(
+        [
+            py() + " -m black " + " ".join(paths),
+            py() + " -m flake8 " + " ".join(paths),
+        ]
+    )
+
+
+def test():
+    args = [
+        '--location="https://otg-novus100g.lbj.is.keysight.com:5000"',
+        (
+            '--ports="otg-novus100g.lbj.is.keysight.com;1;1'
+            " otg-novus100g.lbj.is.keysight.com;1;2"
+            " otg-novus100g.lbj.is.keysight.com;1;3"
+            ' otg-novus100g.lbj.is.keysight.com;1;4"'
+        ),
+        "--ext=ixnetwork",
+        "--speed=speed_100_gbps",
+        "tests/raw",
+    ]
+    run(
+        [
+            py() + " -m pytest -svvv {}".format(" ".join(args)),
```

Comment:
few convergence tests will be added with fixture `dut` , which needs dut for validation. please add "-m not dut"